tensortrade/exchanges/exchange.py

```diff
@@ -229,7 +233,15 @@ def current_price(self, symbol: str) -> float:
         Returns:
             The current price of the specified instrument, denoted in the base instrument.
         """
-        raise NotImplementedError
+        raise NotImplementedError()
+
+    def _bind_trade_price(self, price: float) -> float:
```

Comment:
The `_bind_trade_price` and `_bind_trade_amount` names are confusing to me. Perhaps they should be renamed to `_get_valid_trade_price` and `_get_valid_trade_amount`.